compass/landice/tests/mesh_modifications/subdomain_extractor/subdomain_extractor.cfg

```diff
@@ -0,0 +1,31 @@
+[subdomain]
+
+# path to file from which to extract subdomain
+source_file = TO BE SUPPLIED BY USER
+
+# path to region mask file for source_file
+region_mask_file = TO BE SUPPLIED BY USER
+
+# region number to extract
+region_number = 1
```

Comment:
I think that might overcomplicate the test case for what I imagine is a fairly rare use case.  The test case takes just a couple minutes to run, so even if wanting to process a handful of regions, it should not take too much time to run each region serially.

However, I did add the ability to use a geojson file to specify a custom region.  I kept running into situations where the existing regions had in adequate gutter and the gutter extrapolation wasn't able to produce good results.  This would also be useful for creating larger regions.